js/index.js

```diff
@@ -41,11 +46,18 @@ function updatePriceContainer({ current, points_change: { percent, points } }) {
   const isPositive = points >= 0
   const color = isPositive ? greenColor : redColor
   for (priceChangeEl of priceChangeEls) {
-    priceChangeEl.innerHTML = `${points} (${percent}%)`
-    priceChangeEl.style.color = color
+    if (market_status === 1) {
+      // Market Open
+      priceChangeEl.innerHTML = `${points} (${percent}%)`
+      priceChangeEl.style.color = color
+    } else {
```

Comment:
You could use `elif market_status === 0` if we want to add other market statuses in the future.